_posts/RhymesWithMecca/2016-02-09-BeccaandJames'Chapter8Part2.md

```diff
@@ -1,9 +1,9 @@
 ---
 layout: post
 author:
--RhymesWithMecca
--jamesma560
-title: Becca and James' Chapter 8 Exercises, Part 2
+- RhymesWithMecca
+- jamesma560
+title: Becca and James' Chapter 8 Exercises Part 2
```

Comment:
The spaces are good but I think you'll need double quotes around this title on account of the apostrophe.  Also check the apostrophe in your filename.

Good job sticking with it!  I know it's hard to get all the pieces right when the feedback cycle is so slow.